src/xavier/gui.py

```diff
@@ -283,11 +283,7 @@ def launch_gui(DEBUG=True):
                 genome=genome,
                 targets=values["-TARGETS-"]
                 if values["-TARGETS-"]
-                else (
-                    xavier_base("resources", "Agilent_SSv7_allExons_hg38.bed")
-                    if genome == "hg38"
-                    else(xavier_base("resources", "SureSelect_mm10_sorted.bed"))
-                ),  # TODO should this be part of the genome config file?
+                else (""),  # TODO should this be part of the genome config file?
```

Comment:
The way I have designed the GUI, the option to change the targets file only appears when the user clicks additional settings and even, the default button is "default targets". When the user clicks the custom radio button, then the layout to choose a file shows up. Check the GUI [screenshots](https://ccbr.github.io/XAVIER/latest/usage/gui/#35-additional-settings) for details.
So, i guess to answer your question, the values["-TARGETS-"] will only be entered by the user if they jump through all these hoops, else they will not enter any value to values["-TARGETS-"]. Also, I am not sure if the GUI itself can store default values, it just creates variables to store values which pops up when user selects additional settings. But you can check the gui.py code if I am wrong or mistaken.

Reply:
Yes, I think we should find out what happens if a user clicks the custom radio button but then enters nothing for the targets field. Will there be an error or will something such as `None` or `""` be assigned to `values["-TARGETS-"]`?

Reply:
It throws an error message saying "Custom Targets BED file selected but not provided!!"

Reply:
ok perfect! Then there's no need for this if-else.

Based on this code chunk, since you explicitly test for `values["-TARGETS-"] == ""` that tells me that the GUI assigns an empty string to it.

https://github.com/CCBR/XAVIER/blob/edd9a3e097fbb4be7694b19299e81a2faf1b2f6f/src/xavier/gui.py#L260-L265

Reply:
By the way, you can simplify this line further

https://github.com/CCBR/XAVIER/blob/edd9a3e097fbb4be7694b19299e81a2faf1b2f6f/src/xavier/gui.py#L261

to just 
```python
if values["-TARGETS-"]:
```

Because `bool("")` evaluates to `False`.

Reply:
I removed line 284-285 and updated it to `targets=values["-TARGETS-"],` as suggested below, and also updated line 261 to be just `if values["-TARGETS-"]:`

but now if I run the gui with selecting custom gui with no value, it doesn't throw an error and still uses the default targets bed file.

Reply:
@samarth8392 sorry I made a typo in my suggestion in that if-else statement, it should have been:

```python
if not values["-TARGETS-"]:
```